frontend/cypress/integration/accounts-list-page.spec.ts

```diff
@@ -51,8 +51,8 @@ context("Accounts List page", () => {
       .should("exist")
       .first()
       .click();
-    cy.get(".account-info-container", { timeout: 5000 }).should("exist");
-    cy.get(".account-info-container .card-cell .card-body")
+    cy.get(".c-dOsIRQ", { timeout: 5000 }).should("exist");
+    cy.get(".c-dOsIRQ .card-cell .card-body")
```

Comment:
Are CSS identifiers stable across builds? Do changes in the CSS rules change the identifier? I can see how that could be bugging experience to update many tests after you edited CSS rules.

Reply:
Identifiers are stable across builds, as they are based on hashing styles themselves. Changing the style will change the hash.
We'll probably have more readable names [soon](https://github.com/modulz/stitches/pull/916), and we'll be able to rely not on full name (which is now `c-<hash>`), but on what name starts with (`c-My-Element-<hash>`).